docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md

```diff
@@ -13,14 +13,14 @@ sidebar_position: 3
 
 The **LSP1UniversalReceiverDelegateUP** is a contract called by the **[`universalReceiver(...)`](./lsp0-erc725-account.md#universalreceiver)** function of the **[LSP0ERC725Account](./lsp0-erc725-account.md)** contract that:
 
-- Writes the keys representing **[LSP7-DigitalAsset](./lsp7-digital-asset.md)** and **[LSP8-IdentifiableDigitalAsset](./lsp8-identifiable-digital-asset.md)** assets received into the account storage, and remove them when the balance is zero according to the **[LSP5-ReceivedAssets Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)**.
+- Writes the keys representing the assets received from the **[LSP7-DigitalAsset Standard](./lsp7-digital-asset.md)** and **[LSP8-IdentifiableDigitalAsset Standard](./lsp8-identifiable-digital-asset.md)** into the account storage and removes them when the balance is zero according to the **[LSP5-ReceivedAssets Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)**.
```

Comment:
The same remarks of the vault go here, the assets are not received from the lsp7/lsp8 standards.